perfact/zodbsync/tests/test_sync.py

```diff
@@ -2152,3 +2153,36 @@ def test_layer_update_warn(self, caplog):
             assert expect + '/Test' in caplog.text
             assert 'AttributeError' not in caplog.text
             assert expect + '/ToDelete/Sub' in caplog.text
+
+    def test_layer_frozen(self):
+        """
+        Verify that changed files are properly written into the custom
+        layer in case the layer below is frozen.
+        """
+        with self.runner.sync.tm:
+            self.app.manage_addProduct['OFSP'].manage_addFile(id='blob')
+
+        with self.addlayer(frozen=True) as layer:
```

Comment:
Pretty sure that `frozen = True` is the default for layers below the top-most custom layer. So not sure why this was necessary, but it still is OK.